xml/admin_monitoring_alerting.xml

```diff
@@ -1,1058 +1,374 @@
-
-# The VictorOps API URL.
-[ api_url: <replaceable>STRING</replaceable> | default = global.victorops_api_url ]
-
-# A key used to map the alert to a team.
-routing_key: <replaceable>TMPL_STRING</replaceable>
-
-# Describes the behavior of the alert (one of 'CRITICAL', 'WARNING', 'INFO').
-[ message_type: <replaceable>TMPL_STRING</replaceable> | default = 'CRITICAL' ]
-
-# Summary of the alerted problem.
-[ entity_display_name: <replaceable>TMPL_STRING</replaceable> | default = '{{ template "victorops.default.entity_display_name" . }}' ]
-
-# Long explanation of the alerted problem.
-[ state_message: <replaceable>TMPL_STRING</replaceable> | default = '{{ template "victorops.default.state_message" . }}' ]
-
-# The monitoring tool the state message is from.
-[ monitoring_tool: <replaceable>TMPL_STRING</replaceable> | default = '{{ template "victorops.default.monitoring_tool" . }}' ]
-
-# Configuration of the HTTP client.
-[ http_config: <replaceable>HTTP_CONFIG</replaceable> | default = global.http_config ]
-</screen>
-   </example>
-   <example xml:id="alert-webhook">
-    <title><replaceable>WEBHOOK_CONFIG</replaceable></title>
-    <para>
-     You can use the webhook receiver to configure a generic receiver.
-    </para>
-<screen>
-# Whether or not to notify about resolved alerts.
-[ send_resolved: <replaceable>BOOLEAN</replaceable> | default = true ]
-
-# The endpoint for sending HTTP POST requests.
-url: <replaceable>STRING</replaceable>
-
-# Configuration of the HTTP client.
-[ http_config: <replaceable>HTTP_CONFIG</replaceable> | default = global.http_config ]
-</screen>
-    <para>
-     &alertmanager; sends HTTP POST requests in the following JSON format:
-    </para>
-<screen>
-{
- "version": "4",
- "groupKey": <replaceable>STRING</replaceable>, // identifycation of the group of alerts (to deduplicate)
- "status": "&lt;resolved|firing>",
- "receiver": <replaceable>STRING</replaceable>,
- "groupLabels": <replaceable>OBJECT</replaceable>,
- "commonLabels": <replaceable>OBJECT</replaceable>,
- "commonAnnotations": <replaceable>OBJECT</replaceable>,
- "externalURL": <replaceable>STRING</replaceable>, // backlink to Alertmanager.
- "alerts": [
-   {
-     "status": "&lt;resolved|firing>",
-     "labels": <replaceable>OBJECT</replaceable>,
-     "annotations": <replaceable>OBJECT</replaceable>,
-     "startsAt": "&lt;rfc3339>",
-     "endsAt": "&lt;rfc3339>",
-     "generatorURL": <replaceable>STRING</replaceable> // identifies the entity that caused the alert
-   },
-   ...
- ]
-}
-</screen>
-    <para>
-     The webhook receiver allows for integration with the following
-     notification mechanisms:
-    </para>
-    <itemizedlist>
-     <listitem>
+    <note>
       <para>
-       DingTalk (https://github.com/timonwong/prometheus-webhook-dingtalk)
+        The &prometheusmgr; needs to be restarted for any of the following
+        configuration changes to be applied.
```

Comment:
Because of the phrase "the following config changes" I was expecting this to be followed by a list.

As it isn't followed by a list, can we drop the word "following"? Or perhaps replace it with "any of these config changes"?